ibis-server/pyproject.toml

```diff
@@ -62,6 +62,7 @@ markers = [
 [tool.ruff]
 line-length = 88
 target-version = "py311"
+exclude = ["tools/"]
```

Comment:
I ignored the ruff for the scripts. I think it's not for production purpose. It would be more flexible.

Reply:
Hmmm. Having it would be a plus, but it's not essential. Go ahead.